<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskForm</class>
 <widget class="QWidget" name="TaskForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>237</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="taskNameLabel">
         <property name="text">
          <string>Nom</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="taskName"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="subTaskLabel">
         <property name="text">
          <string>Sous-tâche de</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="subTask"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="priorityLabel">
         <property name="text">
          <string>Priorité</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="priority"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="timeLimitLabel">
         <property name="text">
          <string>Date limite</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QRadioButton" name="timeLimitChoiceAbsolute">
         <property name="text">
          <string>Fixe</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QRadioButton" name="timeLimitChoiceRelative">
         <property name="text">
          <string>Relative</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QDateEdit" name="timeLimitAbsolute"/>
       </item>
       <item row="6" column="1">
        <layout class="QFormLayout" name="formLayout_3">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QSpinBox" name="timeLimitRelative"/>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="timeLimitRelativeLabel">
           <property name="text">
            <string>Jours</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="acceptTask">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
       <property name="centerButtons">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
